kotlinc/src/main/java/org/jetbrains/kotlin/cli/jvm/compiler/jarfs/FastJarFileSystem.kt

```diff
@@ -0,0 +1,131 @@
+/*
+ * This file is part of Cosmic IDE.
```

Comment:
didn't you remove the original license and replace it with Cosmic IDE's?

Reply:
~~copied the module from cosmic ide, it was auto edited/added by AS~~

Reply:
restored the original license header on files that haven't been modified from source